workflows/src/main/kotlin/hk/edu/polyu/af/bc/badge/flows/badgeclass/CreateBadgeClass.kt

```diff
@@ -36,14 +36,14 @@ class CreateBadgeClass(
      * checkpoint is reached in the code. See the 'progressTracker.currentStep' expressions within the call() function.
      */
     companion object {
-        object GENERATING_TRANSACTION : ProgressTracker.Step("Generating transaction based on new BadgeClass.")
```

Comment:
The progress trackers are not necessary for now. You can just leave it.

Reply:
Done